# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/bindings/Bindings.conf
# Commit: b848ca90888e
# Full Hash: b848ca90888ee8e446f3d54b65c3ef68c71a1451
# Author: longsonr <longsonr@gmail.com>
# Date: 2019-06-20 22:06:31
# Regressor Bug: 1551030
# File Overlap Count: 2
# Description:
#   Bug 1551030 - Merge all SVGRect classes r=dholbert
# ==============================================================================

diff -r 20956a783e08 -r b848ca90888e dom/bindings/Bindings.conf
--- a/dom/bindings/Bindings.conf	Thu Jun 20 14:19:57 2019 +0300
+++ b/dom/bindings/Bindings.conf	Thu Jun 20 07:03:54 2019 -0700
@@ -1005,10 +1005,6 @@
     'headerFile': 'mozilla/dom/SVGGradientElement.h',
 },
 
-'SVGRect': {
-    'nativeType': 'mozilla::dom::SVGIRect'
-},
-
 'SVGStringList': {
     'nativeType': 'mozilla::DOMSVGStringList',
     'headerFile': 'DOMSVGStringList.h',